infra/es_cluster.tf

```diff
@@ -391,6 +391,27 @@ resource "google_project_iam_member" "es-feed" {
   member  = "serviceAccount:${google_service_account.es-feed.email}"
 }
 
+resource "google_project_iam_custom_role" "es-feed-write" {
+  role_id     = "es_feed_write"
```

Comment:
Noob question: how do you decide when to use underscores and when to use dashes?

Reply:
GCP has rules. Some fields accept only dashes, some fields accept only underscores, some fields accept both. It's a bit... arcane.